rust/hyperlane-core/src/utils.rs

```diff
@@ -30,6 +30,12 @@ pub fn domain_hash(address: H256, domain: impl Into<u32>) -> H256 {
     )
 }
 
+/// Converts an H256 to a H160
+pub fn h256_to_h160(h256: H256) -> H160 {
```

Comment:
Is there not a way to convert from H256 -> H160 already? Or does it grab the wrong bytes?
https://docs.rs/primitive-types/latest/primitive_types/struct.H160.html#impl-From%3CH256%3E-for-H160

Reply:
I'm honestly not sure why I did this..